.gitignore

```diff
@@ -2,4 +2,13 @@
 .idea
 build
 *.iml
-application-local.propertie
+application-local.properties
```

Comment:
I think you meant not to check this file in? anyway, I left it checked it but fixed what I think was a misspelling.